hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -159,6 +159,16 @@ public final class OzoneConfigKeys {
   public static final String
       OZONE_METADATA_STORE_ROCKSDB_CF_WRITE_BUFFER_SIZE_DEFAULT = "128MB";
 
+  public static final String OZONE_METADATA_STORE_ROCKSDB_BLOCK_CACHE_SIZE =
+      "ozone.metastore.rocksdb.block.cache.size";
+  public static final String
+      OZONE_METADATA_STORE_ROCKSDB_BLOCK_CACHE_SIZE_DEFAULT = "256MB";
```

Comment:
The default needs to be larger, 1 or 2 GB perhaps?

Reply:
Indeed, this parameter should be larger. For larger tables, 1 to 2 GB is a suitable default value. I will update this parameter.  Hi,  @errose28  What do you think about it?

Reply:
That seems fine to me. I don't have data on the size of serialized proto values but that could help approximate the number of keys the cache would hold.

Reply:
@kerneltime  The default size has been updated to 1GB, thanks for your comment